pip/wheel.py

```diff
@@ -44,6 +44,20 @@
 logger = logging.getLogger(__name__)
 
 
+class Cache(object):
+    """A cache of wheels for future installs."""
+
+    def __init__(self, cache_dir):
+        """Create a wheel cache.
+
+        :param cache_dir: The root of the cache.
+        """
+        self._cache_dir = cache_dir
+
+    def cached_wheel(self, link):
```

Comment:
We could, but its a very one-shot trick :) - I tend to only use it when I need something to substitute in for an existing external give-me-a-function interface.